Sprint-3/implement/get-angle-type.js

```diff
@@ -1,27 +1,24 @@
-// Implement a function getAngleType
+function getAngleType(angle) {
+    if (angle === 90) {
+        return "Right angle";
+    } else if (angle > 0 && angle < 90) {
+        return "Acute angle";
+    } else if (angle > 90 && angle < 180) {
+        return "Obtuse angle";
+    } else if (angle === 180) {
+        return "Straight angle";
+    } else if (angle > 180 && angle < 360) {
+        return "Reflex angle";
+    } else {
+        return "Invalid angle";
+    }
+}
 
-// Acceptance criteria:
+// Example usage:
```

Comment:
There don't seem to be any assertions in here. I think the goal here is to practice writing TDD style, where you write the test first, then just enough code to pass the test, then a new test.

This systematic approach to problem solving is the underlying goal of this sprint, so it's important to try it.

Reply:
A concise approach to TDD for the getAngleType function:
Write tests first:
javascript
Copy code
function testGetAngleType() {
    console.assert(getAngleType(45) === "acute");
    console.assert(getAngleType(90) === "right");
    console.assert(getAngleType(120) === "obtuse");
    console.assert(getAngleType(180) === "straight");
    console.log("All tests passed!");
}